lib/mayaUsd/fileio/utils/adaptor.h

```diff
@@ -136,10 +145,15 @@ class UsdMayaAttributeAdaptor
         const UsdMayaPrimReaderArgs& args,
         UsdMayaPrimReaderContext&    context);
 
+#if PXR_VERSION < 2308
     /// Gets the defining spec for this attribute from the schema registry.
     /// Returns a null handle if this attribute adaptor is invalid.
     MAYAUSD_CORE_PUBLIC
     const SdfAttributeSpecHandle GetAttributeDefinition() const;
+#else
+    MAYAUSD_CORE_PUBLIC
+    UsdPrimDefinition::Attribute GetAttributeDefinition() const;
+#endif
```

Comment:
The old version could return a null, invalid attr spec. Is the new API similar except it got a IsValid() instead of being pointer-like?

Reply:
It has a `bool()` operator that is used to check validity